src/osw/express.py

```diff
@@ -31,8 +34,11 @@
 DOWNLOAD_DIR_DEFAULT = BASE_PATH / "osw_files" / "downloads"
 
 DEPENDENCIES = {
-    # "Entity": "Category:Entity",  # depends on nothing
+    # "Entity": "Category:Entity",  # depends on nothing#
+    "Category": "Category:Category",  # depends on Entity
+    "Property": "Category:Property",  # depends on Entity
```

Comment:
Adding Category and Property to make sure that the namespace and title functions in osw.model.static are working as expected.
@SimonStier: Should we create a mechanism to ensure installed dependencies other than in osw.express?